.erb-lint.yml

```diff
@@ -27,3 +27,7 @@ linters:
         Enabled: false
       Metrics/LineLength:
         Max: 289
+      Rails/OutputSafety:
+        Enabled: true
+        Exclude:
+          - '**/app/views/internal/articles/_individual_article.html.erb'
```

Comment:
this is currently disabled in `.rubocop_todo.yml`, don't think we should enable it here.

It's not a great solution but in the meantime once you verify `html_safe` is the only error shown by the erb linter, you can bypass it with `git commit --no-verify`

Reply:
Got it! I didn't know about `--no-verify` flag, thanks.